sw/device/silicon_creator/lib/drivers/otbn.h

```diff
@@ -45,13 +43,42 @@ typedef enum otbn_status {
   kOtbnStatusLocked = 0xFF,
 } otbn_status_t;
 
+/**
+ * Error codes for the OTBN driver.
+ */
+typedef enum otbn_error_t {
+  /** No errors. */
+  kErrorOtbnOk = 0,
+  /** Invalid argument provided to OTBN interface function. */
+  kErrorOtbnInvalidArgument = (1 << 0),
```

Comment:
Is there a particular reason you're going for bit flag values rather than sequential values? It's fine if there's a reason but we aught right it down.

Reply:
No, no good reason. I thought the `lib/runtime` OTBN driver used bit-flag values like this and was trying to match it, but taking a second look just now I see that it uses sequential values. I'll change these to be sequential as well.